### ISSUE TITLE
Integrate revmc jit

### ISSUE THREAD

**Issue by onbjerg** [ithacaxyz/odyssey#25]
See https://github.com/paradigmxyz/alphanet/issues/86#issuecomment-2377980590

**Comment by 0xurb** [comment]
@onbjerg Can I take it?

**Comment by 0xurb** [comment]
Also seems that `revmc` has older Reth version, which gives me different primitives for several types (such like `SpecId`, `reth_revm::Context`'s `Host` implementation, etc.)

**Comment by DaniPopes** [comment]
Sorry this is already being worked on. And no the types are the same,  it might be due to different revm versions

**Comment by 0xurb** [comment]
Yes, I meant same types, but different versions. As it WIP, I’ll find some other issue to deal with, thanks to respond.
